pkg/config/labels/instance.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	DNS1123LabelMaxLength = 63 // Public for testing only.
-	dns1123LabelFmt       = "[a-zA-Z0-9](?:[-a-zA-Z0-9]*[a-zA-Z0-9])?"
+	dns1123LabelFmt       = "[a-z0-9](?:[-a-z0-9]*[a-z0-9])?"
```

Comment:
Unlike the k8s, we are using this to validate Kind, DNS 1123 can contain upper cases

https://github.com/kubernetes/kubernetes/pull/39675/files

Reply:
@hzxuzhonghu Thanks